src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,30 @@
+import {NgModule} from '@angular/core';
+import {AuthRoutingModule} from './auth-routing.module';
+import {LoginComponent} from './pages/login/login/login.component';
+import {ReactiveFormsModule} from '@angular/forms';
+import {MatFormFieldModule} from '@angular/material/form-field';
```

Comment:
it would be better to create separate module for Material library (which will be import and export needed components from material) and use it in shared module